src/__tests__/tools.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import renderer from "react-test-renderer";
+
+import {
+    EmptyToolListingTest,
+    ErroredListingTest,
+    ToolListingTest,
+} from "../../stories/tools/Listing.stories";
+import { mockAxios } from "../../stories/axiosMock";
+
+beforeEach(() => {
+    mockAxios.reset();
+});
+
+afterEach(() => {
+    mockAxios.reset();
+});
+
+test("Tool Table View renders", () => {
+    const component = renderer.create(<ToolListingTest />);
+    let tree = component.toJSON();
+    expect(tree).toMatchSnapshot();
```

Comment:
I thought we wanted to move away from snapshots and use `chromatic` instead?
So in my last PR I simply rendered a component and unmounted it, to ensure it wouldn't crash in either phase.
```suggestion
    component.unmount();
```

Reply:
I thought that matching the snapshot was using chromatic. 😕 I could easily be mistaken, though. I'm still a newbie when it comes to UI work.

Reply:
I think chromatic automatically creates its own snapshots and rendered images based on the stories that are added in a PR.

My last PR #108 didn't include any snapshot files for the components I created, but chromatic still rendered all the stories I added.

Reply:
Chromatic automates snapshot creation and comparison. No need to manually create and update snapshot with each change 🎉

Reply:
Ah, okay. Thanks. I'll update the code.